mmv1/api/type.go

```diff
@@ -335,6 +335,11 @@ func (t *Type) SetDefault(r *Resource) {
 		if t.Description == "" {
 			t.Description = fmt.Sprintf("A reference to %s resource", t.Resource)
 		}
+	case t.IsA("Fingerprint"):
+		// Represents a fingerprint.  A fingerprint is an output-only
+		// field used for optimistic locking during updates.
+		// They are fetched from the GCP response.
+		t.Output = true
```

Comment:
Should output be set true only if t.Output is nil ?

Reply:
Probably. I can't imagine why someone would explicitly set Output: false, but I guess it could happen